courses/admin.py

```diff
@@ -17,15 +17,15 @@ class CourseRunInline(admin.StackedInline):
 
 class CourseAdmin(admin.ModelAdmin):
     """ModelAdmin for Courses"""
-    list_display = ('position_in_program', 'title',)
+    list_display = ('title', 'position_in_program',)
```

Comment:
given that we are touching this, I would add the program in this list and the modify the ordering below to be first by program and then by position in program.